docker-compose.yml

```diff
@@ -27,6 +27,10 @@ services:
       - redis
     environment:
       - SAMPLES_BUCKET_NAME=czbiohub-idseq-samples-development
+      - IDSEQ_AIRBRAKE_PROJECT_ID
```

Comment:
We can make some changes to idseq_infra and airbrake.rb to take advantage of this.